axelor-bank-payment/src/main/java/com/axelor/apps/bankpayment/service/bankorder/file/BankOrderFileService.java

```diff
@@ -114,6 +118,9 @@ protected String getFolderPath() throws AxelorException {
           paymentMode.getName());
     }
 
+    String dataExportDir = appService.getDataExportDir();
+    folderPath = dataExportDir + paymentMode.getBankOrderExportFolderPath();
```

Comment:
No need to create a variable for dataExportDir
Simply, return appService.getDataExportDir() + folderPath;